integration-tests/mongodb-client/src/test/java/io/quarkus/it/mongodb/BookResourceTest.java

```diff
@@ -1,8 +1,7 @@
 package io.quarkus.it.mongodb;
 
 import static io.restassured.RestAssured.get;
-import static org.hamcrest.Matchers.containsInAnyOrder;
-import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.*;
```

Comment:
wildcard imports are prohibited

Reply:
not sure why IDEA formatted it in this way even I use quarkus formatter. I can change it manually.

Regarding `default` - 1.9.0.Final returns `default` not `__default__` (see [source](https://github.com/quarkusio/quarkus/blob/1.9.0.Final/extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/health/MongoHealthCheck.java#L50) ). That's why I kept `default` in my commit. But changing it to `<default>` it would be minor breaking change.

Reply:
> I wonder if we should provides two health checks instead of one, when reactive health check will lands we may split synchrone and reactive client health checks.

It makes sense to me. But I'll leave decision on you folks.